code/datums/wires/airlock.dm

```diff
@@ -68,15 +68,14 @@
 	if(isliving(usr) && A.hasPower() && A.isElectrified())
 		if (A.shock(usr, 100))
 			return
-	if(A.hasPower()) //Multitool has no effect at all if the door has lost power
+	switch(wire)
+		if(WIRE_POWER1, WIRE_POWER2) // Pulse to lose power, or reset the delay before restoring power if already lost
+			A.loseMainPower()
+		if(WIRE_BACKUP1, WIRE_BACKUP2) // Pulse to lose backup power, or reset the delay before restoring power if already lost
+			A.loseBackupPower()
```

Comment:
Should probably document that pulsing the lose power wire will now make it lose power even when unpowered meaning you can restart the timer now which makes hacking easier

Reply:
I don't follow how this makes hacking easier, the timer being possible to reset has negligible effect on hacking? It can buy you more time before the power comes back on, but hacking takes nowhere near as long as the initial timer anyway, even for someone fairly inexperienced. 

Will document it anyway though.